src/app/lib/utilities/getUrlHelpers/getRecommendationsUrl/index.js

```diff
@@ -1,11 +1,9 @@
-export default ({ assetUri }) =>
-  `${process.env.RECOMMENDATIONS_ENDPOINT}/recommendations${assetUri}`;
-
-export const portugueseRecommendationsExperimentEndpoint = ({
-  assetUri,
-  engine,
-  engineVariant,
-}) =>
-  engineVariant
-    ? `${process.env.RECOMMENDATIONS_ENDPOINT}/recommendations${assetUri}?Engine=${engine}&EngineVariant=${engineVariant}`
-    : `${process.env.RECOMMENDATIONS_ENDPOINT}/recommendations${assetUri}?Engine=${engine}`;
+export default ({ assetUri, engine, engineVariant }) => {
+  if (engineVariant) {
+    return `${process.env.RECOMMENDATIONS_ENDPOINT}/recommendations${assetUri}?Engine=${engine}&EngineVariant=${engineVariant}`;
+  }
+  if (engine) {
+    return `${process.env.RECOMMENDATIONS_ENDPOINT}/recommendations${assetUri}?Engine=${engine}`;
+  }
+  return `${process.env.RECOMMENDATIONS_ENDPOINT}/recommendations${assetUri}`;
```

Comment:
This just makes sure it doesn't provide an `EngineVariant` if no `Engine` is provided as it doesn't make a lot of sense. That shouldn't happen though really.

```suggestion
  let queryParams = '';
  if (engine) {
    queryParams += `Engine=${engine}`;
    if(engineVariant) {
      queryParams += `&EngineVariant=${engineVariant`;
    }
  }
  return `${process.env.RECOMMENDATIONS_ENDPOINT}/recommendations${assetUri}${queryParams}`;
```